integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/pogeneration/JavaPageObjectPrinter.kt

```diff
@@ -0,0 +1,157 @@
+package to.etc.domui.webdriver.core.pogeneration
+
+import org.eclipse.jdt.annotation.NonNullByDefault
+import to.etc.domui.webdriver.core.WebDriverConnector
+import to.etc.domui.webdriver.core.base.BasePO
+import to.etc.util.StringTool
+import java.lang.IllegalStateException
+import java.lang.StringBuilder
+import java.util.stream.Collectors
+
+class JavaPageObjectPrinter(genClassModel: GeneratedClassModel) : PageObjectPrinter {
+	private val sb = StringBuilder()
+	private val m_actual: GeneratedClassModel
+	private val imports = HashSet<String>();
+
+	init {
+		m_actual = GeneratedClassModel(genClassModel.name, genClassModel.namespace, ArrayList(genClassModel.members), ArrayList(genClassModel.methods))
+		val toRemove = ArrayList<GeneratedClassMember>()
+		genClassModel.members.forEach{
+			if(it.accessModifier == POAccessModifier.PRIVATE) {
+				m_actual.members.add(GeneratedClassMember("m_${it.name}", it.type, it.accessModifier, it.params))
+				toRemove.add(it)
+				m_actual.methods.add(GeneratedClassMethod("get${StringTool.strCapitalizedIntact(it.name)}", it.type, POAccessModifier.PUBLIC, "return m_${clean(it.name)};"))
+			}
+		}
+		m_actual.members.removeAll(toRemove)
+	}
+
+	override fun print(): String {
+		appendPackage()
+		appendImports()
+		sb.append("\n")
+		appendClassDeclaration()
+		sb.append("\n")
+		appendMembers()
+		sb.append("\n")
+		appendConstructor()
+		sb.append("\n")
+		appendMethods()
+		sb.append("}\n")
+
+		return sb.toString()
+	}
+
+	private fun actualClassName(): String {
+		return "${m_actual.name}PO"
+	}
+
+	private fun appendConstructor() {
+		sb.append("\tpublic ${actualClassName()}(WebDriverConnector wd) { \n \tsuper(wd); \n\t }")
```

Comment:
There are spaces before the \n and the \t. This means any reformat of this code will bring changes.